install/pre-installation.md

```diff
@@ -47,17 +47,31 @@ Any steps run on an `external` server require that server to have the following
    > - **If air-gapped or behind a strict firewall**, then the tarball must be obtained from the medium delivered by Cray-HPE. For these cases, copy or download the tarball to the working
    >   directory and then proceed to the next step. The tarball will need to be fetched with `scp` during the [Download CSM tarball](#21-download-csm-tarball) step.
 
+1. (`external#`) This step should ONLY be performed if an http proxy is required to access a public endpoint on the internet for the purpose of downloading artifacts. CSM does NOT support the use of proxy servers for anything other than downloading artifacts from external endpoints. The http proxy variables must NOT be exported and must be unset after the desired artifacts are downloaded. Failure to unset the http proxy variables after downloading artifacts will cause many failures in subsuquent install steps.
```

Comment:
This step and the step below (which show up as steps 2, 3, and 4 in the rendered output) look like they should actually be substeps of step one above (i.e. "Download the CSM software release").   Right now they are at the same level.